assemblers/org.osbuild.qemu

```diff
@@ -418,13 +433,11 @@ def grub2_write_boot_image(boot_f: BinaryIO,
     # latter can find the former. To exact location is
     # taken from grub2's "boot.S":
     #  GRUB_BOOT_MACHINE_KERNEL_SECTOR 0x5c (= 92)
-    image_f.seek(0x5c)
+    image_f.seek(0x5C)
```

Comment:
That seems odd, the trend for the last umpteen years has been to use lower-case hex. I used to prefer all upper case hex (my assembly background leaking through) and have finally started getting used to lower case... :)